core/src/test/java/kafka/test/junit/ClusterTestExtensions.java

```diff
@@ -190,6 +190,9 @@ private void processClusterTest(ExtensionContext context, ClusterTest annot, Clu
         }
 
         Properties properties = new Properties();
+        for (ClusterConfigProperty property : defaults.serverProperties()) {
```

Comment:
this is unrelated to this PR, but could we update `config.serverProperties` instead of creating a temporary `properties`?

Reply:
Updated it. Thank you.